packages/shared/src/components/modals/BootPopups.tsx

```diff
@@ -30,15 +30,14 @@ export const BootPopups = (): null => {
   const [bootPopups, setBootPopups] = useState(() => new Map());
   const { getMarketingCta } = useBoot();
   const marketingCta = getMarketingCta(MarketingCtaVariant.Popover);
-  const { shouldShowPopup: shouldShowStreaksPopup } = useStreakExperiment();
-  const { streak, isEnabled } = useReadingStreak();
+  const { streak, shouldShowPopup: shouldShowStreaksPopup } =
+    useReadingStreak();
```

Comment:
Do we need it called here if it is not used?

Reply:
Which in particular are you referring to? the ones we have now are all used

Reply:
My bad, it was just the formatting. Looked like `useReadingStreak` is called alone on new line.